cmd/sqlflowserver/main.go

```diff
@@ -55,7 +56,19 @@ func start(datasource, modelDir string) {
 		}
 	}
 
-	s := grpc.NewServer()
+	var s *grpc.Server
+	if caCrt != "" && caKey != "" {
+		creds, err := credentials.NewServerTLSFromFile(caCrt, caKey)
+		if err != nil {
+			log.Fatalf("failed to load CA crt/key files: %s, %s, %v", caCrt, caKey, err)
+		}
+		s = grpc.NewServer(grpc.Creds(creds))
+		log.Println("Launch server with SSL/TLS certification.")
+	} else {
+		s = grpc.NewServer()
+		log.Println("Launch server with insecure mode.")
+	}
+
```

Comment:
It might make the code more readable if we move L59-L71 into a new function `newServer(caCrt, caKey string) *grpc.Server`.

Reply:
Done.